dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/TrackerPreheat.java

```diff
@@ -630,6 +630,26 @@ public void addProgramOwner( String teiUid, String programUid,
         }
     }
 
+    public void addUsers( Set<User> users )
+    {
+        Map<String, User> userMap = users.stream()
```

Comment:
should we filter out null values? to prevent an NPE in `getUserByUid`? Or is it intentional to store null values to say for example username `foo` there is no user